services/QuillLMS/app/controllers/api/v1/session_feedback_histories_controller.rb

```diff
@@ -1,8 +1,7 @@
 class Api::V1::SessionFeedbackHistoriesController < Api::ApiController
-  # GET /feedback_histories.json?page=1&activity_id=33
+  # GET /feedback_histories.json?page=1&activity_id=33&start_date=2021-04-18T03:00:00.000Z&end_date=2021-05-18T03:00:00.000Z
   def index
-    records = FeedbackHistory.list_by_activity_session(**params.permit(:page, :activity_id).symbolize_keys)
-
+    records = FeedbackHistory.list_by_activity_session(**params.permit(:page, :activity_id, :start_date, :end_date).symbolize_keys)
```

Comment:
I love that you used `permit` here.  A common Rails pattern is to create a function on the controller called `permitted_params` or some such so that you can approve the same params for any action, but this works just fine.